.eslintignore

```diff
@@ -10,5 +10,5 @@ static/vendor/javascripts
 static/src/javascripts-legacy/projects/common/utils/atob.js
 static/src/javascripts-legacy/projects/common/utils/picturefill.js
 
-# tests #karma-jest
```

Comment:
we're planning to remove karma in favour of jest, and `#karma-jest` is a flag we're added to keep track of things to remove/update – probably useful to keep this here?